src/containers/guest-home-page/WelcomeBlock.js

```diff
@@ -1,7 +1,7 @@
 import { useTranslation } from 'react-i18next'
 import { Typography, Box, Button } from '@mui/material'
 
-import { WelcomeBlockStyles as style } from '~/containers/guest-home-page/styles/welcomeBlockStyles'
+import { welcomeBlockStyles as style } from '~/containers/guest-home-page/styles/welcome-block.styles'
```

Comment:
maybe it's better to import immediately as a style

Reply:
Fixed.